project/app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,164 @@
+    <string name="preferencesTwitter" translatable="false">Twitter</string>
+    <string name="preferencesTwitterSummary" translatable="false">twitter.com/owntracks</string>
+    <string name="preferencesEncryptionKeySummary">Clé symétrique pour le chiffrement de la charge utile</string>
+    <string name="preferencesEncryptionKey">Clé de cryptage</string>
+    <string name="preferencesEncryptionKeyDialogMessage">La clé de chiffrement est coupée après 32 caractères.</string>
+    <string name="na">Indisponible</string>
+    <string name="save">"Sauvegarder"</string>
+    <string name="cancel">"Annuler"</string>
+    <string name="preferencesCleanSessionEnabled">Vider la session</string>
+    <string name="publish">Envoyer</string>
+    <string name="preferencesPubExtendedDataSummary">Inclure des données étendues dans les rapports de localisation</string>
+    <string name="preferencesPubExtendedData">Données étendues</string>
+    <string name="preferencesAutostartSummary">Démarrer automatiquement au démarrage de l\'appareil</string>
+    <string name="preferencesAutostart">Démarrage auto</string>
+    <string name="preferencesRemoteCommand">Commandes à distance</string>
+    <string name="preferencesRemoteCommandSummary">Activer les commandes à distance</string>
+    <string name="description">Description</string>
+    <string name="longitude">Longitude</string>
+    <string name="latitude">Latitude</string>
+    <string name="radius">Rayon</string>
+    <string name="addWaypoint">Ajouter un POI</string>
+    <string name="exportConfiguration">Exporter</string>
+    <string name="contactLocationUnknown">Aucun emplacement n\'est disponible</string>
+    <string name="menuContactNavigate">Naviguer</string>
+    <string name="menuCurrentLoction">Localisation actuelle</string>
+    <string name="contactDetailsAccuracy">Précision</string>
+    <string name="contactDetailsTopic">Identifiant</string>
+    <string name="waypointListPlaceholder">Aucune région définie</string>
+    <string name="importConfig">importer</string>
+    <string name="tls" translatable="false">TLS</string>
+    <string name="preferencesSecurity">Sécurité</string>
+    <string name="accept">J\'accepte</string>
+    <string name="preferencesIdentification">Identification</string>
+    <string name="preferencesParameters">Paramètres</string>
+    <string name="preferencesKeepalive" translatable="false">Keepalive</string>
+    <string name="seconds">Secondes</string>
+    <string name="preferencesProfileId">Mode</string>
+    <string name="mode_mqtt_private_label">MQTT privé</string>
+    <string name="aLocation">Région</string>
+    <string name="events">Événements</string>
+    <string name="title_activity_regions">Régions</string>
+    <string name="title_activity_region">Région</string>
+    <string name="title_activity_map">Map</string>
+    <string name="title_activity_contacts">Copains</string>
```

Comment:
Copains are more like "Pals". Should we prefer "Amis" as friends?

Reply:
Agreed